paddle/fluid/pybind/tensor_py.h

```diff
@@ -13,7 +13,13 @@ See the License for the specific language governing permissions and
 limitations under the License. */
 
 #pragma once
+
+#ifndef NPY_NO_DEPRECATED_API
```

Comment:
Why add this? Try to add some comments on code?

Reply:
removed (since we use pybind interface now)